server/apps/main/templates/main/what_autism_is.html

```diff
@@ -40,6 +40,8 @@ <h5 class="text-info" id="about-autspaces"><em></em>About Autism</h5>
       </div>
       <div class="about-autism">
         <h5 class="text-info" id="platform-usage"><em></em>About Sensory Processing</h5>
+        <p>Autistic people often experience the world differently to non-autistic people because our senses work differently. This is not a flaw, it is just a difference, but because so many places have been built by non-autistic people without autistic people in mind, they are often not suitable for us, and may be unpleasant or inaccessible.</p> 
```

Comment:
```suggestion
        <p>Autistic people often experience the world differently to non-autistic people because of senses working differently. This is not a flaw, it is just a difference. But because so many places have been built by non-autistic people without autistic people in mind, they are often not suitable for us, and may be unpleasant or inaccessible.</p> 
```